### ISSUE TITLE
Invalid URLs returned for FileStore-uploaded resources

### ISSUE THREAD

**Issue by seanh** [ckan/ckanext-deadoralive#19]
For resources with files uploaded to CKAN's FileStore we're returning URLs like: `"http:///en/storage/f/2012-11-12T140744/2011-12-31.txt "`

**Comment by seanh** [comment]
This happens when CKAN's site URL is not set. Arguably a bug in CKAN not in ckanext-deadoralive.

**Comment by jpmckinney** [comment]
I think the link checker should respect relative URLs, assuming the URL in CKAN's API is `/en/storage/f/2012-11-12T140744/2011-12-31.txt` and not `http:///en/storage/f/2012-11-12T140744/2011-12-31.txt`

**Comment by seanh** [comment]
No, I think CKAN is actually returning `http:///en/storage/f/2012-11-12T140744/2011-12-31.txt`
